hermes/internal/notification/message/main.go

```diff
@@ -24,7 +24,7 @@ import (
 	"hermes/internal/notification/messageexecutionhistory"
 	"hermes/internal/notification/payloads"
 	"hermes/pkg/errors"
-	"hermes/queueprotocol"
+	"hermes/rabbitClient"
```

Comment:
```suggestion
	"hermes/rabbitclient"
```

Should it be cammel case?